sdk/formrecognizer/Azure.AI.FormRecognizer/tests/Infrastructure/FormRecognizerTestEnvironment.cs

```diff
@@ -61,91 +45,47 @@ public FormRecognizerTestEnvironment() : base("formrecognizer")
         public string TargetResourceRegion => GetRecordedVariable(TargetResourceRegionEnvironmentVariableName);
 
         /// <summary>
-        /// The name of the directory where the running assembly is located.
+        /// The absolute path of the directory where the running assembly is located.
         /// </summary>
-        /// <value>The name of the current working directory.</value>
+        /// <value>The absolute path of the current working directory.</value>
         private static string CurrentWorkingDirectory => Path.GetDirectoryName(Assembly.GetExecutingAssembly().Location);
 
         /// <summary>
-        /// The relative path to the JPG file which contains the form to be used for tests.
-        /// </summary>
-        /// <value>The relative path to the JPG file.</value>
-        public static string FormPath => CreatePath(FormFilename);
-
-        /// <summary>
-        /// The URI string to the JPG file which contains the form to be used for tests.
+        /// Creates an absolute path to a file contained in the local test assets folder.
         /// </summary>
-        /// <value>The URI string to the JPG file.</value>
-        public static string FormUri => CreateUri(FormFilename);
+        /// <param name="filename">The name of the file to create a path to.</param>
+        /// <returns>An absolute path to the specified file.</returns>
+        public static string CreatePath(string filename) =>
+            Path.Combine(CurrentWorkingDirectory, AssetsFolderName, filename);
 
         /// <summary>
-        /// The relative path to the JPG file which contains the receipt to be used for tests.
+        /// Creates a URI string to a file contained in the test assets folder stored in
+        /// the azure-sdk-for-net GitHub repo.
         /// </summary>
-        /// <value>The relative path to the JPG file.</value>
-        public static string JpgReceiptPath => CreatePath(JpgReceiptFilename);
+        /// <param name="filename">The name of the file to create a URI string to.</param>
+        /// <returns>A URI string to the specified file.</returns>
+        public static string CreateUriString(string filename) =>
+            string.Format(FileUriFormat, AssetsFolderName, filename);
```

Comment:
I'm calling the string URI a `UriString`, so we can use the `CreateUri` method to return actual `Uri` instances. I think these names are reasonable given that the [string parameter in the Uri constructor](https://docs.microsoft.com/en-us/dotnet/api/system.uri.-ctor?view=netcore-3.1#System_Uri__ctor_System_String_) is called `uriString`.